parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DALStatement.g4

```diff
@@ -30,3 +30,32 @@ dbLinkInfo
 explain
     : EXPLAIN PLAN (SET STATEMENT_ID EQ_ stringLiterals)? (INTO (schemaName DOT_)? tableName (AT_ dbLinkInfo)? )? FOR (insert | delete | update | select)
     ;
+
+parameterName
+    : identifier
+    ;
+
+showErrors
+    : SHOW ERRORS
+    ;
+
+showParameter
+    : SHOW (PARAMETER | PARAMETERS) parameterName
+    ;
+
+show
+    : showErrors
+    | showParameter
+    ;
+
+fileExt
+    : DOT_ identifier
+    ;
+
+spoolFileName
```

Comment:
Can you support spool statement in next pr?

Reply:
yes i can.